src/metadata/controllers/metadataController.ts

```diff
@@ -145,7 +157,32 @@ export class MetadataController {
     return metadata;
   }
 
-  private async checkValuesValidation(payload: IPayload): Promise<void> {
+  private async validateClassification(classification: string): Promise<boolean | string> {
+    const classifications: string[] = await this.lookupTables.getClassifications();
+    if (classifications.includes(classification)) {
+      return true;
+    }
+    return 'classification is not a valid value!';
+  }
+
+  private async validatePatchValues(payload: IUpdatePayload): Promise<void> {
+    //Validate that the classification is in the possible (from lookup tables)
+    try {
+      if (payload.classification != undefined) {
+        const result = await this.validateClassification(payload.classification);
+        if (typeof result == 'string') {
+          throw new BadValues('classification is not a valid value!');
+        }
+      }
+    } catch (error) {
+      if (error instanceof BadValues) {
+        throw new BadValues('classification is not a valid value!');
```

Comment:
Insert also the valid values for more informative error log. You can look at ingestion-service for an example.
Do it also in validatePostValues